src/Package/Impl/Resources.resx

```diff
@@ -764,7 +764,10 @@ Additional information:
   <data name="HelpSearchWatermark" xml:space="preserve">
     <value>Type search term</value>
   </data>
-  <data name="VariableExplorer_ErrorEnvironment" xml:space="preserve">
-    <value>Error retrieving environment list</value>
+  <data name="Settings_EvaluateActiveBindings" xml:space="preserve">
+    <value>Evalaute active bindings.</value>
```

Comment:
Spelling. Also, since this is the name of the property in the property grid, it shouldn't have the trailing dot.